bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -173,6 +184,33 @@
     final Counter lacUpdateMissesCounter;
     private final OpStatsLogger clientChannelWriteWaitStats;
 
+    /**
+     * Read ahead related parameters
+     */
+    // Number of read ahead entries allowed in cache.
+    private final int maxNumReadAheadEntries = 0;
```

Comment:
can we group all of these fields into a separate datastructure ?
all of these fields are useful only in case of read ahead feature enabled